consensus/spos/bls/errors.go

```diff
@@ -4,3 +4,9 @@ import "errors"
 
 // ErrNilSentSignatureTracker defines the error for setting a nil SentSignatureTracker
 var ErrNilSentSignatureTracker = errors.New("nil sent signature tracker")
+
+// ErrorCreateAndSendSignMessage defines an error for sendSignature function
+var ErrorCreateAndSendSignMessage = errors.New("false sent by createAndSendSignMessage")
```

Comment:
won't be needed if my suggestion would be applied. Also, in order to keep all errors consistent, this should have looked something like `var ErrCreateAndSendSignMessage = errors.New("createAndSendSignMessage failed")`

Reply:
removed the errors with the new implementation